src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+
+ 		/**
+ 		 * Method to click on New Sub Order Order button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 	public void clickNewSubOrderButton() throws IOException {
+ 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 		log.info("click on New Sub Order button.");
+ 		Assert.assertTrue(clickNewSubOrderButton.isDisplayed());
+ 		clickNewSubOrderButton.click();
+ 		JavaScriptExec.sleep();
+ 		}
+ 		
+ 	   @FindBy(how=How.XPATH, using="//*[@id='ui-id-6']")
+ 		//*[@id="ui-id-6"]
+ 	//*[@id="ui-id-6"]
+ 	 private WebElement clickDetailsTab;
+ 	    /**
+ 	     * Method to Click Products tab.
+ 	     * @throws IOException 
+ 	     */
+ 	  public void clickDetailsTab() throws IOException{
+ 	    CreateOrderPage sp=new CreateOrderPage(driver);
+ 	    log.info("Click Details Tab");
+ 	    Assert.assertTrue(clickDetailsTab.isDisplayed());
+ 	    clickDetailsTab.click();
+ 	    JavaScriptExec.sleep();
+ 	    }
+ 	    
+ 	 @FindBy(how=How.XPATH, using="//*[@id='order-details-form']/div[1]/div[1]/span/a")
+ 	//*[@id="order-details-form"]/div[1]/div[1]/span/a
+	 private WebElement clickNew;
+	    /**
+	     * Method to Click Products tab.
+	     * @throws IOException 
+	     */
+	  public void clickNew() throws IOException{
+	    CreateOrderPage sp=new CreateOrderPage(driver);
+	    log.info("Click Details Tab");
+	    Assert.assertTrue(clickNew.isDisplayed());
+	    clickNew.click();
+	    JavaScriptExec.sleep();
+	    }
+ 	    
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--11-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--11-update-form"]/div[2]/a[1]/span
+ 	
+ 	 	private WebElement clickUpdateButton2;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton2() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton2.isDisplayed());
+ 	 		clickUpdateButton2.click();
+ 	 		JavaScriptExec.sleep();
+ 	 			}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6-update-form']/div[2]/a[1]")
+ 	 	//*[@id="change--6-update-form"]/div[2]/a[1]/span
+ 	 
+ 	 
+ 	
+ 	 	private WebElement clickUpdateButton3;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void clickUpdateButton3() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("click on Update button.");
+ 	 		Assert.assertTrue(clickUpdateButton3.isDisplayed());
+ 	 		clickUpdateButton3.click();
+ 	 		JavaScriptExec.sleep();
+ 	 			}
+ 	 		
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--9-update-form']/div[2]/a[1]")
+ 	 //*[@id="change--9-update-form"]/div[2]/a[1]/span
+ 	 //*[@id="change--9-update-form"]/div[2]/a[1]/span
+ 	 
+ 	 	private WebElement clickUpdateButton4;
+
+ 	 		/**
+ 	 		 * Method to click on Update button.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
```

Comment:
@mayankchhajer Please remove the xpaths after pasting and format the code

Reply:
Done